Makefile

```diff
@@ -362,6 +362,7 @@ stylesheets-check: generate-stylesheets
 .PHONY: generate-stylesheets
 generate-stylesheets:
 	node_modules/.bin/lessc --clean-css public/less/index.less public/css/index.css
+	node_modules/.bin/postcss --use autoprefixer --autoprefixer.browsers "> 2%, last 2 firefox versions, last 2 safari versions" -o public/css/index.css public/css/index.css
 	$(foreach file, $(filter-out public/less/themes/_base.less, $(wildcard public/less/themes/*)),node_modules/.bin/lessc --clean-css public/less/themes/$(notdir $(file)) > public/css/theme-$(notdir $(call strip-suffix,$(file))).css;)
```

Comment:
should postcss be run on the theme files too?

Reply:
Yes it should, fixed.

Reply:
Thanks @silverwind! Could you also change the calls to `node_modules/.bin/` binaries to an `npx` call. That way we don't need to reference the specific binaries in node_modules. (ex. `node_modules/.bin/lessc` becomes `npx lessc`)

Reply:
That would require a recent version of npm (I think 5 and up). Are we okay with this? The way we have it is more compatible.

Reply:
You’re right. Compatibility is better.